### PR TITLE
Remove files from common

### PR DESCRIPTION
**Pull request by CharliePoole**
This is a partial fix toward #1596. It removes Env.cs and XmlHelper.cs from common.

### CODE REVIEW

**Comment by CharliePoole** [comment]
This is ready to merge once the builds pass.

**Comment by CharliePoole** [comment]
@nunit/core-team I'm building another PR on top of this one, so it will be helpful is someone can review and merge.

File: src/Common/nunit/CommandLineOptions.cs [hunk 1]
```diff
@@ -66,9 +75,9 @@ public CommandLineOptions(params string[] args)
                 Parse(args);
         }
         
-        #endregion
+#endregion
         
-        #region Properties
+#region Properties
```
Comment (rprouse) [inline src/Common/nunit/CommandLineOptions.cs]: All the regions got unindented. This happens with VS2015 whenever you add new `#if` statements. Should we just accept the changes until VS gets its act together?

**Comment by rprouse** [comment]
Looks good except for my one question on the regions. Make a call, I am good either way. Fix it, or we accept it.

File: src/Common/nunit/CommandLineOptions.cs (hunk 1 above)
Comment (CharliePoole) [inline src/Common/nunit/CommandLineOptions.cs]: Let's not worry about it for now. This is the first of a series of changes that will hit this file anyway. I'll try to clean them all up in #1621

**Pull request merged**